x/mongo/driver/mongocrypt/mongocrypt_test.go

```diff
@@ -124,9 +125,13 @@ func testKmsCtx(t *testing.T, ctx *Context, keyAltName bool) {
 	kmsCtx := ctx.NextKmsContext()
 	hostname, err := kmsCtx.HostName()
 	noerr(t, err)
-	expectedHost := "kms.us-east-1.amazonaws.com:443"
-	if hostname != expectedHost {
-		t.Fatalf("hostname mismatch; expected %s, got %s", expectedHost, hostname)
+
+	// on MacOS and Linux, hostname will include the default port "443". On Windows,
```

Comment:
```suggestion
	// Only check for the hostname. libmongocrypt versions that do not include MONGOCRYPT-352 does not include the port.
```

I suggest updating this comment to clarify this is not platform specific, but rather inconsistency between libmongocrypt versions.

Reply:
Good point; updated to something similar.